draft-packages/menu/KaizenDraft/Menu/components/MenuContent.module.scss

```diff
@@ -90,13 +90,22 @@ $padding: $kz-var-spacing-xs;
     }
   }
 
+  &:focus-visible,
+  &.focus-visible {
+    border: 2px $kz-var-color-cluny-500 solid;
```

Comment:
Small improvement here could be to only change the border colour to avoid having two separate references of `2px` and `solid`

Reply:
```suggestion
    border-color: $kz-var-color-cluny-500;
```